src/HybridRedisCache/HybridCacheServiceCollectionExtensions.cs

```diff
@@ -23,8 +24,15 @@ public static IServiceCollection AddHybridRedisCaching(this IServiceCollection s
         var options = new HybridCachingOptions();
         setupAction(options);
 
+
+        var redisConnection =
+            ConnectionMultiplexer.Connect(options.ConvertHybridCacheOptionsToRedisOptions());
+
+        services.AddSingleton<IConnectionMultiplexer>(redisConnection);
```

Comment:
Please don't add IConnectionMultiplexer as singleton. Because, some applications need to run on multiple Redis databases as parallel and this make an issue.